### DISCUSSION TITLE
GPDR checklist

### DISCUSSION THREAD

**Discussion by evilcel3ri** [tosdr/edit.tosdr.org#1043] [Ideas]
Inspired from [here](https://github.com/tootsuite/mastodon/issues/7280)

* [ ] updated version of our ToS @hugoroy 
* [x] allow user to delete their data (still a work in progress, I've started something there but hasn't finished it) @piks3l #335 
* [ ] allow user to export all its data (maybe we could make it a cool way to export user's point and contributions ?)

**Comment by michielbdejong** [comment]
I'm unassigning myself to indicate that I'm not currently working on this issue.
